Create new home pages

ZzRanger opened the pull request:
## Overview

Created designs for new home pages. You can see the changes [here](https://nebula-planner-fm7pa2rmh-utdnebula.vercel.app/). Below are the mockups for the new home pages.  Feedback on the pages would be greatly appreciated. I also created 2 versions of the profile page.Please let me know in the comments which profile page is preferred.

![Profile Page](https://user-images.githubusercontent.com/47403443/194783096-8b062476-6e7c-4434-949d-0df2678b545d.png)

![Plans](https://user-images.githubusercontent.com/47403443/194783094-edc370eb-073a-453c-9364-5409b571d4a7.png)

ZzRanger committed:
feat: Implement new home page

ZzRanger committed:
chore: Update home page UI

ZzRanger committed:
chore: Add Inter font

ZzRanger committed:
feat: Create 1 profile page view

ZzRanger committed:
feat: Create second profile page view

ZzRanger committed:
chore: Touch up Profile2

ZzRanger committed:
feat: Create initial implementation of drawer

ZzRanger committed:
chore: Touch up home page

ZzRanger committed:
chore: Set default route to new home pages

This is a temporary measure; will revert this before merging

ZzRanger committed:
chore: Add padding to Profile2

1solomonwakhungu commented:
On smaller screens, the menu should be hidden or darken the background when the menu is open on smaller screens. Attached is what the a person sees on a smaller screen. 

![Screenshot_20221009-203708.jpg](https://user-images.githubusercontent.com/65043605/194789256-9233a215-d151-4b25-bfed-7512b040c6cd.jpg)

1solomonwakhungu commented:
Profile picture is too big on mobile screens. 

![Screenshot_20221009-203915.jpg](https://user-images.githubusercontent.com/65043605/194789331-2748acee-2fe4-4d3e-b2f5-99f615385e8b.jpg)

jasonappah commented:
+1 on Solomon's comments, a few other nits I wanted to mention. 

- The sidebar items and input field labels are still set in Roboto instead of Inter.
- The main header text, and sidebar text/icons aren't using the right color - they should be a dark blue instead of black (#1C2A6D).
- None of the sidebar items seem to have a selected state.

This is looking great, really happy to see this coming together.

jasonappah commented on components/newhome/Home.tsx:
```diff
@@ -0,0 +1,73 @@
+          className="col-span-full text-white rounded-xl w-32 h-12 flex justify-center items-center flex-row bg-[#3E61ED]"
+        >
+          <div className="flex flex-row ">
+            <div className="text-3xl flex flex-col justify-center items-center">
```
```suggestion
            <div className="text-3xl flex flex-col justify-center items-center mr-2">
```
Adding a bit more margin on the icon. Not looking to have the button look exactly how it does in Figma, but I feel the icon needs a little more room to breathe.

innub commented on components/newhome/Profile.tsx:
```diff
@@ -0,0 +1,101 @@
+      </section>
+      <section className="mt-40 flex flex-col gap-y-8 self-center items-center">
+        <article className="bg-white">
+          <div className="grid grid-cols-2 gap-x-32 gap-y-16 px-6 py-4">
```
If we are considering mobile screens, this container should probably change to a flex column on smaller screens. We should probably design mobile screens on the Figma beforehand though.

ZzRanger committed:
chore: Update banner on profile 2

aanos-m commented:
First of all, very very happy with the progress. 
So on my end, I think we should add some padding between the "Delete" button and the message before it. I feel like it just looks like it was just mushed together. 

![Capture](https://user-images.githubusercontent.com/106491552/195492205-e69c4ecd-3331-48e3-ac28-d62e07a7ec7d.PNG)

Secondly, on the credits page, I feel like we should add some spacing between the header and the Profile div/box and see how that looks and make both the "Profile" box/div and the "Delete My Account" div/box a little rounded on the edges like we have done for the other boxes on the Home and Profile page.

ZzRanger committed:
chore: Address PR comments for home & profile

ZzRanger committed:
chore: Update default project styles

ZzRanger committed:
chore: Add media query hook

This React hook returns whether or not the window is currently above a certain
size. The hook is useful for creating responsive UI

ZzRanger committed:
chore: Update styles for home pages

ZzRanger committed:
feat: Add ability to update name, change email, and reset password

ZzRanger committed:
chore: Integrate home pages into Planner

ZzRanger committed:
chore: Minor changes

ZzRanger committed:
Merge remote-tracking branch 'origin/develop' into feature/new-home-pages

ZzRanger committed:
chore: Merge credits into home pages

ZzRanger committed:
chore: Move drawer UI into its own component

ZzRanger reviewed (commented):
Quick notes about profile page: I disabled the "Email" field since updating email requires at the very least some re-authentication in order to authorize such an action. We don't have that logic implemented yet. I also disabled the "major" and "minor" fields since we don't use that information anywhere else in Planner.

ZzRanger committed:
chore: Integrate onboarding into user profile

ZzRanger committed:
chore: Small style changes

ZzRanger committed:
chore: Fix scroll css

ZzRanger committed:
chore: Change background color of Credits page
